lms/templates/help_modal.html

```diff
@@ -208,7 +202,7 @@
     cycle_modal_tab("#feedback_submit", "#feedback_form_wrapper .close-modal");
     cycle_modal_tab("#feedback-faq-link", "#feedback_success_wrapper .close-modal");
 % else:
-    cycle_modal_tab("#howto-enroll-link", "#help_wrapper .close-modal");
+    cycle_modal_tab("#last-link", "#help_wrapper .close-modal");
```

Comment:
Is there existing CSS or some other logic for handling `#last-link`? I'm curious about the significance.